bin/mail/2016-03-21-teacher-recruitment/generate-district-teachers

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env ruby
+require_relative '../mailing-common/mailing-list-utils'
+require 'set'
+
+# Teachers in all regions from February email, minus NYC and Chicago
+
+# Exported from https://docs.google.com/spreadsheets/d/1T_cd7CDlOuuLe_hObSzNX7BbN7PwU_EzEpjuIHuwoNM
+REGION_LIST_FILE = 'Code.org Partners with Regions - Sheet1.csv'
+SKIP_REGIONS = ['New York', 'Chicago']
+
+# https://www.dropbox.com/s/upxdp1yam0jd9lf/1.2.2016.csv?dl=0
+CURRENT_TEACHERS_FILE = 'current-teachers.csv'
+EXCLUDED_EMAILS = Set.new
+REGIONS = {}
+
+def query_with_location(base_query, city, state)
+  query = "(#{base_query}) && ((location_state_s:\"#{state}\" && location_city_s:\"#{city}\") || (create_ip_state_s:\"#{state}\" && create_ip_city_s:\"#{city}\"))"
```

Comment:
Reformat to the 80 character maximum line length. Also in other places throughout this PR.